src/main/java/org/sopt/confeti/api/performance/controller/PerformanceController.java

```diff
@@ -144,9 +147,14 @@ public ResponseEntity<BaseResponse<?>> searchAutoComplete(
     public ResponseEntity<BaseResponse<?>> getRecommendPerformanceId(
             @UserId(require = false) Long userId
     ) {
-        RecommendMusicsPerformanceDTO recommendMusicsDTO = performanceFacade.getRecommendPerformanceId(userId);
-        return ApiResponseUtil.success(SuccessMessage.SUCCESS,
-                RecommendMusicsPerformanceResponse.from(recommendMusicsDTO));
+        Optional<RecommendMusicsPerformanceDTO> performanceDto = Objects.isNull(userId)
+                ? performanceFacade.getRecommendPerformanceIdByRand()
+                : performanceFacade.getRecommendPerformanceIdByUserId(userId);
+
+        return performanceDto
+                .map(performance ->
+                        ApiResponseUtil.success(SuccessMessage.SUCCESS, RecommendMusicsPerformanceResponse.from(performance)))
+                .orElseGet(()-> ApiResponseUtil.success(SuccessMessage.SUCCESS, Collections.emptyMap()));
```

Comment:
```suggestion
        return ApiResponseUtil.success(SuccessMessage.SUCCESS, 
            performanceDto.map(RecommendMusicsPerformanceResponse::from)
                .orElseGet(Collections.emptyMap())
        );
```

이렇게 작성하는건 어떠신가요?
중복된 코드를 줄일 수 있을 것 같아요!